README.md

```diff
@@ -393,6 +395,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 * [Hoa EventSource](https://github.com/hoaproject/Eventsource) - An event source library.
 * [Evenement](https://github.com/igorw/evenement) - An event dispatcher library.
 * [Event](https://github.com/thephpleague/event) - An event library with a focus on domain events.
+* [Event](https://github.com/cakephp/event) - An event dispatcher library (part of CakePHP).
```

Comment:
Could you pick an acronym for CakePHP, something short (see the other frameworks for examples), and put in brackets after CakePHP (e.g. `(CP)`) in frameworks, and then replace all the `(part of CakePHP)` bits with it too. Thanks.